kiwmi/desktop/xdg_shell.c

```diff
@@ -388,10 +388,11 @@ xdg_shell_view_get_size(
     uint32_t *width,
     uint32_t *height)
 {
-    struct wlr_box *geom = &view->xdg_surface->geometry;
+    struct wlr_box geom;
+    wlr_xdg_surface_get_geometry(view->xdg_surface, &geom);
```

Comment:
Using `view->geom` may be a better idea. I re-ask for it only because the previous version also does.